pkg/azure/servicebus/subscription_count_client.go

```diff
@@ -0,0 +1,79 @@
+package servicebus
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/servicebus/mgmt/2017-04-01/servicebus"
```

Comment:
I currently use the Azure SDK for Go as it allowed for the retrieval of each type of count within `CountDetails`. In the current case of `azure-service-bus-go` this number can be skewed if there are letters in the DLQ not being pulled

Reply:
https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues